### ISSUE TITLE
Unencrypted PK -> Encrypted Keystore File

### ISSUE THREAD

**Issue by tayvano** [MyEtherWallet/MyEtherWallet#311]
See https://github.com/kvhnuke/etherwallet/issues/1163

Users often want to go from their raw private key to an encrypted keystore file. Right now, we have them use our Chrome Extension of via geth command line. As the use of the Chrome Extension will change moving forward, we need to have this functionality in MEW somewhere. 

The user would need to provide their unencrypted private key and a password and out would pop a keystore file. 

Not sure if this is already done or being worked on, but wanted to get it in writing somewhere so it doesn't get missed.

Not a v4 blocker, just needs to be done before we deprecate Chrome Extension functions.

💖

**Comment by HenryNguyen5** [comment]
Internally the code is already there to handle this, would just need the UI portion!

**Comment by HenryNguyen5** [comment]
@eddiewang 
Code for doing this lives in `ethereumjs-wallet` itself, you'll need to make an instance of a wallet via `fromPrivateKey` then call `.toV3()` on the wallet instance.

**Comment by eddiewang** [comment]
#336
